README.adoc

```diff
@@ -497,12 +506,17 @@ A unique name for this rule.
 The `Gemfile` which Bundler runs with.
 
 |`gemfile_lock` a|
```

Comment:
Should this be now called `gemfile`? Since we compute the lock file automatically?

Reply:
You are right. I'll fix it so that it still respects `gemfile_lock` when it's passed.

Reply:
Just pushed fixup commit 7f192df0f0ffaaf7825bf11a09790773567fb9ba which addresses this and actually makes less changes to existing. If it's good, I'll rebase and squash the commits together.